packages/components/src/FilterDateRangePicker/subcomponents/FilterDateRangePickerField/types.ts

```diff
@@ -1,26 +1,4 @@
-import React from "react"
-import { FieldMessageStatus } from "@kaizen/draft-form"
-import { StringSuggestions } from "~types/StringSuggestions"
-
-export type { DateRange, DisabledDays } from "@kaizen/date-picker/src/types"
-
-// Ensure you also update the storybook argTypes `locale` options.
-export type FilterDRPSupportedLocales = StringSuggestions<"en-US" | "en-AU">
```

Comment:
Don't we need to keep this? These locales specific to `FilterDRP`; if they are the same as `FilterDP` it's a coincidence

Reply:
Don't both support the same cause it's the same Date Picker package?